src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/LoansToDirectorsServiceImpl.java

```diff
@@ -0,0 +1,95 @@
+package uk.gov.companieshouse.web.accounts.service.smallfull.impl;
+
+import org.apache.commons.lang.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.util.UriTemplate;
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+
+import uk.gov.companieshouse.api.model.accounts.smallfull.loanstodirectors.LoansToDirectorsApi;
+import uk.gov.companieshouse.web.accounts.api.ApiClientService;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.service.smallfull.LoansToDirectorsService;
+import uk.gov.companieshouse.web.accounts.service.smallfull.SmallFullService;
+import uk.gov.companieshouse.web.accounts.validation.helper.ServiceExceptionHandler;
+
+@Service
+public class LoansToDirectorsServiceImpl implements LoansToDirectorsService {
+
+    @Autowired
+    private ApiClientService apiClientService;
+
+    @Autowired
+    private SmallFullService smallFullService;
+
+    @Autowired
+    private ServiceExceptionHandler serviceExceptionHandler;
+
+    private static final UriTemplate LOANS_TO_DIRECTORS_URI =
+            new UriTemplate("/transactions/{transactionId}/company-accounts/{companyAccountsId}/small-full/notes/loans-to-directors");
+
+    private static final String RESOURCE_NAME = "loans to directors";
+
+    @Override
+    public void createLoansToDirectors(String transactionId, String companyAccountsId) throws ServiceException {
+
+        ApiClient apiClient = apiClientService.getApiClient();
+
+        LoansToDirectorsApi loansToDirectors = getLoansToDirectors(apiClient, transactionId, companyAccountsId);
+
+        if (loansToDirectors == null) {
```

Comment:
This logic is duplicated from the controller - we already know `loansToDirectors` is null